python/cudf/cudf/core/column/numerical.py

```diff
@@ -128,6 +129,43 @@ def has_nulls(self, include_nan=False):
             self.nan_count != 0 if include_nan else False
         )
 
+    def __setitem__(self, key: Any, value: Any):
+        """
+        Set the value of ``self[key]`` to ``value``.
+
+        If ``value`` and ``self`` are of different types, ``value`` is coerced
+        to ``self.dtype``. Assumes ``self`` and ``value`` are index-aligned.
```

Comment:
```suggestion
        to ``self.dtype``.
```

In the context of a `Column`, does index alignment need to be mentioned?

Reply:
Nope, was a copy-paste mishap